docs/zh/latest/admin-api.md

```diff
@@ -566,12 +566,12 @@ APISIX 的 Upstream 除了基本的负载均衡算法选择外，还支持对上
 `hash_on` 比较复杂，这里专门说明下：
 
 1. 设为 `vars` 时，`key` 为必传参数，目前支持的 Nginx 内置变量有 `uri, server_name, server_addr, request_uri, remote_port, remote_addr, query_string, host, hostname, arg_***`，其中 `arg_***` 是来自 URL 的请求参数，[Nginx 变量列表](http://nginx.org/en/docs/varindex.html)
-2. 设为 `header` 时, `key` 为必传参数，其值为自定义的 header name, 即 "http\_`key`"
-3. 设为 `cookie` 时, `key` 为必传参数，其值为自定义的 cookie name，即 "cookie\_`key`"。请注意 cookie name 是**区分大小写字母**的。例如："cookie_x_foo" 与 "cookie_X_Foo" 表示不同的 `cookie`。
+2. 设为 `header` 时，`key` 为必传参数，其值为自定义的 header name，即 "http\_`key`"
+3. 设为 `cookie` 时，`key` 为必传参数，其值为自定义的 cookie name，即 "cookie\_`key`"。请注意 cookie name 是**区分大小写字母**的。例如："cookie_x_foo" 与 "cookie_X_Foo" 表示不同的 `cookie`。
 4. 设为 `consumer` 时，`key` 不需要设置。此时哈希算法采用的 `key` 为认证通过的 `consumer_name`。
 5. 如果指定的 `hash_on` 和 `key` 获取不到值时，就是用默认值：`remote_addr`。
 
-以下特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6：为-apache-apisix-构建-openresty)：
+以下特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤-6：为-apache-apisix-构建-openresty)：
```

Comment:
The picture shows broken link because docs' changes still take no effect.

Original link `./how-to-build.md#步骤6：为` is also broken at [github repo](https://github.com/apache/apisix/blob/master/docs/zh/latest/how-to-build.md#%E6%AD%A5%E9%AA%A46%EF%BC%9A%E4%B8%BA-apache-apisix-%E6%9E%84%E5%BB%BA-openresty), but it works at [apisix site](https://apisix.apache.org/zh/docs/apisix/how-to-build#%E6%AD%A5%E9%AA%A46%EF%BC%9A%E4%B8%BA-Apache-APISIX-%E6%9E%84%E5%BB%BA-OpenResty)

Rule `Place one space before/after number`, `./how-to-build.md#步骤6：为` not matched.